pages/common/lldb.md

```diff
@@ -0,0 +1,15 @@
+# lldb
+
+> The LLDB Debugger.
+
+- Debug an executable:
+
+`lldb {{executable}}`
+
+- Attach lldb to running process with id {{PID}}:
+
+`lldb -p {{PID}}`
+
+- Wait for process to launch and attach to it:
+
+`lldb -w -n {{name}}`
```

Comment:
In this case, neither of those processes -- `lldb` will instead wait for a _new_ process to be launched with name `{{name}}`, and then attach to that. (This is incredibly handy when you need to attach to a child process launched by some other process)